src/AWS.Deploy.CLI/Commands/GenerateDeploymentProjectCommand.cs

```diff
@@ -215,7 +215,7 @@ private async Task<bool> IsDirectoryUnderSourceControl(string saveCdkDirectoryPa
         /// <param name="projectDisplayName">The name of the deployment project that will be displayed in the list of available deployment options.</param>
         private async Task GenerateDeploymentRecipeSnapShot(Recommendation recommendation, string saveCdkDirectoryPath, string projectDisplayName)
         {
-            var targetApplicationDirectoryName = _directoryManager.GetDirectoryInfo(_targetApplicationFullPath).Parent.Name;
+            var targetApplicationDirectoryName = _directoryManager.GetDirectoryInfo(_targetApplicationFullPath).Parent!.Name;
```

Comment:
I'd prefer if we do some null handling here instead of suppressing it.

Reply:
Done